README.md

```diff
@@ -1264,6 +1319,35 @@ You can set a timeout for retrying the operations sent to a member. This can be
 
+When invocation is being retried, the client may wait some time before it retries again. This retry wait time is configurable with this configuration property:
+
+```
+config.setProperty(“hazelcast.client.invocation.retry.pause.millis”, “500");
+```
+The default retry wait time is 1 second.
+
+## 7.3.3 Client Backpressure
+
+Hazelcast uses operations to make remote calls. For example, a `map.get` is an operation and a `map.put` is one operation for the primary
+and one operation for each of the backups, i.e. `map.put` is executed for the primary and also for each backup. In most cases, there will be a natural balance between the number of threads performing operations
+and the number of operations being executed. However, there are two situations where this balance and operations
+can pile up and eventually lead to Out of Memory Exception (OOME):
+
+- Asynchronous calls: With async calls, the system may be flooded with the requests.
+- Asynchronous backups: The asynchronous backups may be piling up.
+
+To prevent the system from crashing, Hazelcast provides back pressure. Back pressure works by:
+
+- limiting the number of concurrent operation invocations,
+- periodically making an async backup sync.
+
+Sometimes, e.g., when your servers are overloaded, you may want to slow down the client operations to the cluster. Then the client can be configured to wait until number of outstanding invocations whose reponses are not received to become less than a certain number. This is called "Client Backpressure". By default, the backpressure is disabled. There are a few properties which control the backpressure. These client configuration properties are:
+
+- `hazelcast.client.max.concurrent.invocations`: The maximum number of concurrent invocations allowed. To prevent the system from overloading, you can apply a constraint on the number of concurrent invocations. If the maximum number of concurrent invocations has been exceeded and a new invocation comes in, then Hazelcast will throw HazelcastOverloadException. By default this property is configured as INT32_MAX.
```

Comment:
throw `HazelcastOverloadException`. By default, this property is configured as `INT32_MAX`.